### PR TITLE
ZK-3268: uploader and dropupload progress update interval are different.

### PR DESCRIPTION
**Pull request by klyvechen**

### CODE REVIEW

File: zktest/src/archive/test2/B85-ZK-3268.zul [hunk 1]
```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+B85-ZK-3268.zul
+
+        Purpose:
+                
+        Description:
+                
+        History:
+                Thu May 24 4:58 PM:14 CST 2018, Created by klyve
```
[the remark this answers is not in the document]
Comment (klyvechen) [reply zktest/src/archive/test2/B85-ZK-3268.zul]: ok

File: zktest/src/archive/test2/B85-ZK-3268.zul [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+Copyright (C) 2018 Potix Corporation. All Rights Reserved.
+
+-->
+
+<zk>
+	<label multiline="true">
+		1.clice "Upload Image" button, select the B85-ZK-3268Upload.png, the upload progress bar should be seen.
```
[the remark this answers is not in the document]
Comment (klyvechen) [reply zktest/src/archive/test2/B85-ZK-3268.zul]: ok

File: zktest/src/archive/test2/B85-ZK-3268.zul [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+	<button label="Upload Image" upload="true,maxsize=10000">
+		<attribute name="onUpload"><![CDATA[
+            org.zkoss.util.media.Media media = event.getMedia();
+            if (media instanceof org.zkoss.image.Image) {
+                org.zkoss.zul.Image image = new org.zkoss.zul.Image();
+                image.setContent(media);
+            } else {
+                Messagebox.show("Not an image: "+media, "Error", Messagebox.OK, Messagebox.ERROR);
+                break;
+            }
```
[the remark this answers is not in the document]
Comment (klyvechen) [reply zktest/src/archive/test2/B85-ZK-3268.zul]: ok

File: zktest/src/archive/test2/B85-ZK-3268.zul [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+-->
+
+<zk>
+	<label multiline="true">
+		1.clice "Upload Image" button, give a 1MB ~ 8MB file, the upload progress bar should be seen.
+	</label>
+	<button label="Upload Image" upload="true,maxsize=10000">
+		<attribute name="onUpload"><![CDATA[
+            org.zkoss.util.media.Media media = event.getMedia();
+			Messagebox.show(media, Messagebox.OK, Messagebox.ERROR);
```
Comment (scribetw) [inline zktest/src/archive/test2/B85-ZK-3268.zul]: Error usage, check again.

File: zktest/src/archive/test2/B85-ZK-3268.zul [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+        History:
+                Thu May 24 15:37:21 CST 2018, Created by klyve
+
+Copyright (C) 2018 Potix Corporation. All Rights Reserved.
+
+-->
+
+<zk>
+	<label multiline="true">
+		1.clice "Upload Image" button, give a 1MB ~ 8MB file, the upload progress bar should be seen.
```
Comment (scribetw) [inline zktest/src/archive/test2/B85-ZK-3268.zul]: click

File: zktest/src/archive/test2/B85-ZK-3268.zul [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+	</label>
+	<button label="Upload Image" upload="true,maxsize=10000">
+		<attribute name="onUpload"><![CDATA[
+            org.zkoss.util.media.Media media = event.getMedia();
+			Messagebox.show(media, Messagebox.OK, Messagebox.ERROR);
+        ]]></attribute>
```
Comment (scribetw) [inline zktest/src/archive/test2/B85-ZK-3268.zul]: Check the indentation
```xml
<button label="Upload Image" upload="true,maxsize=10000">
	<attribute name="onUpload"><![CDATA[
	// blah blah blah
        ]]></attribute>
</button>
```

File: zktest/src/archive/test2/B85-ZK-3268.zul (hunk 5 above)
Comment (klyvechen) [reply zktest/src/archive/test2/B85-ZK-3268.zul]: ok

File: zktest/src/archive/test2/B85-ZK-3268.zul (hunk 4 above)
Comment (klyvechen) [reply zktest/src/archive/test2/B85-ZK-3268.zul]: ok

File: zktest/src/archive/test2/B85-ZK-3268.zul (hunk 6 above)
Comment (klyvechen) [reply zktest/src/archive/test2/B85-ZK-3268.zul]: ok

**Comment by wenninghsu** [comment]
@zkadmin rebase

**Comment by zkadmin** [comment]
I just pushed up the changes, enjoy!

**Pull request merged**